authenticationprovider-sample/package.json

```diff
@@ -41,9 +41,9 @@
     "@types/isomorphic-fetch": "^0.0.35",
     "@types/node": "^16.11.7",
     "@types/vscode": "^1.60.0",
-    "@typescript-eslint/eslint-plugin": "^5.30.0",
-    "@typescript-eslint/parser": "^5.30.0",
-    "eslint": "^8.13.0",
+    "@typescript-eslint/eslint-plugin": "^5.42.0",
```

Comment:
Does the lint rule need to change?

Reply:
I only updated the rules ones where we were using `.` since that was resulting in us checking extra files. This package was already using `eslint src --ext ts` which should be ok too